README.md

```diff
@@ -24,6 +24,14 @@ For documentation and reasoning on the rules themselves, see the [Documentation]
 For users upgrading from StyleCop Classic, see [KnownChanges.md](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/tree/master/documentation/KnownChanges.md)
 for information about known differences which you may notice when switching to StyleCop Analyzers.
 
+### C# language versions
+Not all versions of StyleCop.Analyzers support all features of each C# language version. The table below shows the minimum version of StyleCop.Analyzers required for proper support of a C# language version.
+
+| C# version | StyleCop.Analyzers version | Visual Studio version |
+|------------|----------------------------|-----------------------|
+|  1.0 - 6.0 | v1.0.2                     | VS2015+               |
+|  7.0 - 7.3 | v1.1.0-beta                | VS2017                |
```

Comment:
❓ Will this be confusing for VS 2015 users? The 1.1.0 releases continue to work with C# 6 in Visual Studio 2015.

Reply:
I've added ``or higher`` to more clearly indicate that the table contains the minimum version required.